src/UserForm.js

```diff
@@ -131,14 +132,20 @@ class UserForm extends React.Component {
     parentResources: PropTypes.object.isRequired,
     pristine: PropTypes.bool,
     submitting: PropTypes.bool,
+    buttonStyle: PropTypes.string,
```

Comment:
This style is applied to submit button according to the code below. Could you please clarify why the label is named 'submitLabel'  but the put style is named like 'buttonStyle'?